pkg/util/test/req.go

```diff
@@ -57,3 +57,15 @@ func MakeTrace(requests int, traceID []byte) *tempopb.Trace {
 
 	return trace
 }
+
+func MakeTraceWithSpanCount(requests int, spansEach int, traceID []byte) *tempopb.Trace {
```

Comment:
Just curious why we have `requests` and `spansEach` as opposed to a `spansTotal`?

Reply:
I suppose it was what made sense at the time.  Since it's a testing function I'm not super concerned.